src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -83,6 +83,8 @@ class Dpa(object):
     azimuth_range: The radar azimuth range (degrees) as a tuple of
       (min_azimuth, max_azimuth) relative to true north.
     catb_neighbor_dist: The CatB neighbording distance (km).
+    dpa_type: The DPA category, either 'esc' or 'portal'.
+
     grants: The list of registered grants.
```

Comment:
I remove from here. I put underscore because the variable is now "private" (pseudo private, you can always access it in python), simply because it is unsafe to set it directly from some client code (and it may be tempting to do so).
You should instead always pass through some of the function SetGrantsFromFad() or SetGrantsFromList(). This is because when we change the grants we need to do some class state changing things (ResetLists and has_th_grants setup).

I passed the `has_th_grants`  as private, because client code does not need it most likely. It is more of an internal thing to avoid recalculating the state each time.